src/Host/Client/Impl/Definitions/IRCallbacks.cs

```diff
@@ -73,5 +73,8 @@ public interface IRCallbacks {
         /// </summary>
         /// <returns></returns>
         void ViewObject(string expression, string title);
+
+        void PackagesInstalled();
```

Comment:
I though about that, but basically few reasons:

1. We may choose to pass list of packages in the notification so we can process Remove faster as it only removes items from collection.
2. Same in the intellisense package index

I made them the same for now just to save some time. We can optimize package manager and the package index in completion some time later.